frontend/src/metabase/query_builder/components/view/View.jsx

```diff
@@ -44,7 +44,7 @@ import {
   QueryBuilderViewRoot,
   StyledDebouncedFrame,
   StyledSyncedParametersList,
-} from "./View.styled";
+} from "./ViewHeader/View.styled";
```

Comment:
Shouldn't `View.styled.tsx` stay in the same directory as `View.jsx` ?

Reply:
Yep, my bad. Fat fingers :)
Thanks for catching this.